sources/code/renderer/preload/settings.ts

```diff
@@ -91,10 +91,10 @@ function generateSettings(optionsGroups: htmlConfig) {
             });
           } else if(!("dropdown" in setting || "input" in setting || "keybind" in setting)) {
             (Object.keys as keys)(setting).sort().map(key => {
-              if(key !== "name" && key !== "description" && key !== "labels" && setting[key] !== undefined) {
+              if (key !== "name" && key !== "description" && key !== "labels" && setting[key] !== undefined && setting.labels) {
```

Comment:
This line is no longer needed after I updated types, as WebCord will now make sure all of the required types are available. You're free to revert this code.
```suggestion
              if(key !== "name" && key !== "description" && key !== "labels" && setting[key] !== undefined) {
```